test/e2e/manifests/components/base/kustomization.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1alpha1
 kind: Component
 
 resources:
-- ../../../../../manifests/base/crds/minimal
+- ../../../../../manifests/base/crds/full
```

Comment:
This then won't test the minimal CRDs?

Reply:
Nope. There isn't much to test with the minimal CRDs, besides maybe ensuring they validate against the schema.  If you think that's worth testing, I could add a step to each E2E test job that runs `kubectl apply --server-side --validate=strict --dry-run=server --kustomize manifests/base/crds/minimal/`

Reply:
@terrytangyuan Would you like me to enter a PR that adds `kubectl apply --server-side --validate=strict --dry-run=server -f 'manifests/*.yaml'` to validate the release manfiests?

Reply:
Yes please add that step. Thanks!

Reply:
Done! https://github.com/argoproj/argo-workflows/pull/14089